app/components/templates/templates.html

```diff
@@ -6,8 +6,8 @@
   </rd-header-title>
   <rd-header-content>Templates</rd-header-content>
 </rd-header>
-
-<div id="selectedTemplate" class="row" ng-if="state.selectedTemplate">
+<div id="selectedTemplate" />
```

Comment:
I don't really like the introduction of a new div... could you test it further? I've tried the `id="selectedTemplate"` on the row div and it is working well on Chrome and Firefox. On which browser did you have the issue?